server/plugin/api.go

```diff
@@ -216,19 +217,45 @@ func (p *Plugin) completeConnectUserToGoogle(c *Context, w http.ResponseWriter,
 	config := p.getConfiguration()
 
 	code := r.URL.Query().Get("code")
-	if len(code) == 0 {
+	// Length looks to be 73 consistently but we'll check for empty and > 100 just in case because this code is generated by Google.
+	if len(code) == 0 || len(code) > 100 {
```

Comment:
Why `len(code) > 100` is invalid? Also the error message in that case would be missleading.